base/multidimensional.jl

```diff
@@ -863,3 +863,193 @@ If `dim` is specified, returns unique regions of the array `itr` along `dim`.
         @nref $N A d->d == dim ? sort!(uniquerows) : (1:size(A, d))
     end
 end
+
+"""
+    C, ia, ib, ic = uniqueind(itr, dim)
+
+A function that operates similiarly to `unique(itr,dim)` but returns multiple
+output arguments having the following properties:
+
+C - the unique elements of the array `itr` along the selected dimension `dim`
+ia - a Vector{Int} of indices such that:
```

Comment:
Could you provide a general description of the meaning of this vector? Then the examples below will make it more concrete. Without this "and so forth for higher dimensional arrays." is a bit imprecise.